tests/bluetooth/host/buf/bt_buf_get_evt/src/main.c

```diff
@@ -92,14 +92,17 @@ static struct net_buf_pool *get_memory_pool(bool discardable)
  */
 ZTEST(bt_buf_get_evt_default_events_returns_not_null, test_returns_not_null)
 {
-	static struct net_buf expected_buf;
+#define USER_DATA_SIZE sizeof(struct bt_buf_data)
```

Comment:
you could instead define a static inline fn in `zephyr/tests/bluetooth/host/buf/mocks/buf_help_utils.h` to allocate a net_buf with a given size (and set the user_data_size).